zk-chat-server-lib/src/semaphore/api.ts

```diff
@@ -61,9 +58,32 @@ const getMembersForGroup = async (baseUrl: string): Promise<IGroupMember[]> => {
 /**
  * Returns an ordered list of the leaf indexes of removed members in the group.
  */
-const getRemovedMembersForGroup = async (baseUrl: string): Promise<number[]> => {
-    // NOTE: (Kevin) I don't think we need this function for Zuzalu
-    return [];
+const getRemovedMembersForGroup = async (baseUrl: string, id: string): Promise<number[]> => {
+    const url = baseUrl + `/semaphore/${id}`;
+    try {
+        const res = await axios({
+            method: 'GET',
+            timeout: 5000,
+            url,
+            headers: {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+            }
+        });
+        // TODO: ask if this is the right way to detect removed members
+        const removedMembers: number[] = res.data.members
+            .filter((member:string) => member === '00000000000000000000000000000000000000000000000000000000000000000000000000000')
```

Comment:
Since "zero value" is changed to `hash(group_id)` in semaphore group

https://github.com/semaphore-protocol/semaphore/blob/33d814fe6d30b4f2c42fa0e1013e061fe6b3b4ec/packages/group/src/group.ts#L22

To make this filter effective, we should change "00...00" to `hash(id).toString()` too.

```suggestion
            .filter((member:string) => member === hash(id).toString())
```

If we assume there is only group 1, we can just use the `DEFAULT_ZERO_VALUE` for now too
https://github.com/Rate-Limiting-Nullifier/zk-chat-client-server/blob/b586ecfbd92ab079095add305e61f4166c96e35a/server/src/config.ts#L7